exp3/tuning.py

```
@@ -0,0 +1,75 @@
+    all_files = data_utils.get_audio_files()
+    train_files, test_files = data_utils.split_train_test(all_files)
+    total_test = sum(len(f) for f in test_files.values())
+
+    # 3. 构建模板
+    templates = {}
+    for label in config.LABELS:
+        templates[label] = []
+        for path in train_files[label]:
+            try:
+                templates[label].append(features.extract_mfcc(path))
+            except: pass
```

Ruff (0.14.5)

Do not use bare `except`

(E722)